lib/DBIx/Class/Wrapper/Factory.pm

```diff
@@ -146,6 +146,30 @@ sub single {
   return $original ? $self->wrap($original) : undef;
 }
 
+=head2 slice
+
+Equivalent to DBIC Resultset 'slice' method.
+
+See <DBIx::Class::ResultSet/slice>
+
+=cut
+
+sub slice {
+    my ($self, @rest) = @_;
+
+    if ( wantarray ) {
```

Comment:
Can you remove the want array? It's incorrect anyway as far as this factory is concerned ('cause it doesn't return wrapped objects).